### PR TITLE
Update HIP-0002

### PR DESCRIPTION
**Pull request by lukeburns**

### CODE REVIEW

**Review by Falci — APPROVED** [review]

File: HIP-0002.md [hunk 1]
```diff
@@ -45,6 +49,10 @@ In this case, the security would rely on CA. It should be a consumer choice to a
 
 It's also possible to define a non-crypto symbol, such as `/.well-known/wallets/USD`, which should return an [IBAN](https://www.iso.org/standard/41031.html).
 
+### Implementations
+
+- [hip2-dane](https://github.com/lukeburns/hip2-dane)
+
```
Comment (pinheadmz) [inline HIP-0002.md:55]: optional: include https://github.com/pinheadmz/hip2-server ?

File: HIP-0002.md [hunk 2]
```diff
@@ -35,6 +35,10 @@ This proposal doesn't define if the content served should be static (file in the
 
 Requests should be made using only HTTPS. Handshake's HTTPS requests are secured by DANE & PoW.
 
+In implementations that allow sending to both addresses and domain aliases, measures must be taken to disambiguate between addresses and domains of the same length. If sharing an entry field, aliases should be prefixed with an `@` symbol. 
+
+For example, `hs1qshuyulxra3pqpwr40303t8pn79232zztuk4qgz` should be interpreted as a wallet address, while `@hs1qshuyulxra3pqpwr40303t8pn79232zztuk4qgz` should be interpreted as a domain alias with wallet address located at `hs1qshuyulxra3pqpwr40303t8pn79232zztuk4qgz/.well-known/wallets/HNS`.
```
Comment (pinheadmz) [inline HIP-0002.md:40]: IMO, domain names that are valid addresses should simply fail and we should refuse to resolve them. Why not?

**Review by pinheadmz — CHANGES_REQUESTED** [review]

File: HIP-0002.md (hunk 2 above)
Comment (Falci) [reply HIP-0002.md:40]: Agreed.

**Review by Falci — APPROVED** [review]

File: HIP-0002.md (hunk 2 above)
Comment (Falci) [reply HIP-0002.md:40]: Approving as it is, besides the comment, because this how the protocol is currently used (Bob Wallet)

**Pull request merged**